packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/index.stories.tsx

```diff
@@ -4,30 +4,35 @@
 
 import React from 'react';
 import ResetPasswordWithRecoveryKeyVerified from '.';
-import { LocationProvider } from '@reach/router';
+import { createMemorySource } from '@reach/router';
 import { Meta } from '@storybook/react';
 import { withLocalization } from '../../../../.storybook/decorators';
+import {
+  produceComponent,
+  createHistory,
```

Comment:
Getting an error here on `createHistory` import, can just remove.

You can also remove `AppContextValue` and `createMemorySource` since those are unused imports. (Does our new linting pick up unused imports? Are we supposed to run that manually on all PRs or does CI check?)

Reply:
We run the linter in the CI. Looks like a typescript issue killed the the CI pipeline before the linter could finish though.